Main Menu & Level Selecter

JonneRijk opened the pull request:
The basics for the main menu and the level selecter

JonneRijk committed:
Main menu

Working on level selecter

JonneRijk committed:
switching between main menu and level selecter works

JonneRijk committed:
checks for the unlocked levels

checks for unlocked levels when opening the level selecter

JonneRijk committed:
changed build settings

JonneRijk committed:
Merge branch 'development' into Menu/MainMenu

JonneRijk committed:
fixed eventSystem

:WideJoeyHappy:

JonneRijk committed:
works with arrow keys

main menu works with arrow keys

JonneRijk committed:
Added the arrow key selection for the level selecter

And fixed that level 1 is the button selected

JonneRijk committed:
load the level from the level selecter

JonneRijk committed:
Merge branch 'development' into Menu/MainMenu

projojoboy reviewed (changes requested):
Controller loses focus on button when switching between screens

JonneRijk committed:
Added Warning at the start

A warning that the player needs to connect a controller. Also Fixed a bugg with the button selection.
